### PR TITLE
Hello Microverse 

### PR DESCRIPTION
**Pull request by StevenWafeek**
In this project we:
- Add H1 Hello Microverse!
- Add CSS file 
- Add background color style

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+If you like this project...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank...
+
```
Comment (laiifuu) [inline readme.md]: Kindly fill this section by adding some acknowledgments, you can thank your friends, your mentor ...etc. Whoever or whatever you think helped you for building this project.

File: readme.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project...
+
```
Comment (laiifuu) [inline readme.md]: Kindly fill this section as it is a required section in the README. You can ask other to star the project if they like it for example.

File: readme.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
```
Comment (laiifuu) [inline readme.md]: Kindly fill these in with the proper information regarding your project. Considering this is a simple HTML project, I suggest omitting the server and database titles, and for the client, adding HTML and CSS.

File: readme.md [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
```
Comment (laiifuu) [inline readme.md]: Considering you didn't go into much detail in the getting started section, I suggest you remove these 4 lines.

**Review by laiifuu — CHANGES_REQUESTED** [review]
## Hi @StevenWafeek  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
so-close


## Highlights
- [x] Correct linters setup :heavy_check_mark: 

## Required Changes ♻️
- [ ] As mentioned in the [project requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md#:~:text=Add%203%20files,ignored%20by%20git%22.), the HTML file, CSS file, and test.md file, should each be committed in a separate commit. Your commit history does not demonstrate that. 

_Check the comments under the review :point_down:_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by StevenWafeek** [comment]
hello @laiifuu thanks for the feedback 
I made most of them but how do I make the requested change about the commit history?
should I make a new pull request?

**Comment by laiifuu** [comment]
Hey @StevenWafeek :wave:
No problem :v: 
There's no need for a new pull request, you can do the following: 
- Delete the files added (HTML and CSS files) and remove `test.md` from the `gitignore` file. Commit these changes. 
- Add the HTML, CSS and test.md files each in a separate commit.
- Add `test.md` back to the `gitignore` file. After doing this, you'll notice that the file will not be ignored even if its name is inside `gitignore`. This is due to the previous commit and the fact that `gitignore` only ignores `untracked files`. You can check this [link](https://git-scm.com/book/en/v2/Git-Basics-Recording-Changes-to-the-Repository#:~:text=In%20short%2C%20tracked%20files%20are,not%20in%20your%20staging%20area.) for more info.
- Now you need to `untrack` the `test.md` file. I believe this [link](https://stackoverflow.com/questions/1274057/how-do-i-make-git-forget-about-a-file-that-was-tracked-but-is-now-in-gitignore) can help you with this. 
After this you can go ahead and commit again. You'll see that test.md is now being ignored. 

I hope you find this helpful :smile:

**Review by ShadyShawkat — APPROVED** [review]
## Hi @StevenWafeek, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- All linters are passing ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**